com.unity.render-pipelines.core/Runtime/Volume/VolumeComponent.cs

```diff
@@ -30,6 +30,25 @@ public VolumeComponentMenu(string menu)
         }
     }
 
+    /// <summary>
+    /// This attribute allows you to add a button to the editor that links to the documentation.
+    /// Useful for <see cref="VolumeComponent"/> that do not implement a custom <see cref="VolumeComponentEditor"/>.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+    public sealed class VolumeComponentDocumentation : Attribute
```

Comment:
Is the [HelpURL] not usable on VolumeComponent ?
I think it is usable on non GameObject. (I don't remember having to modify it for the refactor and I use it on extension).
Though you need to grab it yourself in this case, but as you do here with [VolumeComponentDocumentation].

Reply:
Good point, swapped in `HelpURL` and removed the redundant `VolumeComponentDocumentation`.